flow/project/cli/config/json/config_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+* Flow CLI
+*
+* Copyright 2019-2020 Dapper Labs, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+package json
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_SimpleJSONConfig(t *testing.T) {
+	b := []byte(`{
+		"emulators": {
+			"default": {
+				"port": 3569,
+				"serviceAccount": "emulator-account"
+			}
+		},
+		"contracts": {},
+		"networks": {
+			"emulator": {
+				"host": "127.0.0.1:3569",
+				"chain": "flow-emulator"
+			}
+		},
+		"accounts": {
+			"emulator-account": {
+				"address": "f8d6e0586b0a20c7",
+				"keys": "11c5dfdeb0ff03a7a73ef39788563b62c89adea67bbb21ab95e5f710bd1d40b7",
+				"chain": "flow-emulator"
+			}
+		},
+		"deploys": {}
+	}`)
+
+	err := ioutil.WriteFile("test1-flow.json", b, os.ModePerm)
+	require.NoError(t, err)
+
+	conf, loadErr := Load("test1-flow.json")
+	os.Remove("test1-flow.json")
```

Comment:
I would add a separate `json.LoadString` function and then remove IO operations from the tests. These can be unreliable in CI environments.

I don't think it's too valuable to be testing the actual IO operations in this case, but if it was, the better solution would be to mock out the filesystem entirely.

Reply:
No, I really don't like my solution but wasn't aware of an alternative - still fresh in go. I will adapt.